src/log.js

```diff
@@ -399,10 +399,11 @@ export class Log {
  * @return {string}
  */
 function toString(val) {
-  if (val instanceof Element) {
+  // Do check equivalent to `val instanceof Element` without cross-window bug
+  if (val.nodeType == 1) {
     return val.tagName.toLowerCase() + (val.id ? '#' + val.id : '');
   }
-  return val;
+  return /** @type {string} */ (val);
```

Comment:
Right. I thought it was `*`. No, in this case, let's keep the closure type cast.